src/Parser/IntlMoneyParser.php

```diff
@@ -32,6 +32,7 @@ public function __construct(\NumberFormatter $formatter)
      */
     public function parse($money, $forceCurrency = null)
     {
+        $currency = null;
         $decimal = $this->formatter->parseCurrency($money, $currency);
```

Comment:
And it also has [`NumberFormatter::parse`](http://php.net/manual/en/numberformatter.parse.php).